lib/tangles/get-tangle-updates.js

```diff
@@ -9,7 +9,16 @@ const B_CONTENT = Buffer.from('content')
 const B_TANGLES = Buffer.from('tangles')
 const B_ROOT = Buffer.from('root')
 
-module.exports = function getUpdates(ssb, tangle, root, cb) {
+function getTangleUpdates(ssb, tangle, root, cb) {
+  pull(
+    tangleUpdateStream(ssb, tangle, root),
+    pull.collect(cb)
+  )
+}
+getTangleUpdates.stream = tangleUpdateStream
+module.exports = getTangleUpdates
```

Comment:
I just tweaked this file so you can get _EITHER_ a callback _OR_ a stream. :wink:

Reply:
aah that's what the .stream is for i was confused